src/components/routerLink.browser.spec.ts

```diff
@@ -11,6 +10,7 @@ import { visibilityObserverKey } from '@/compositions/useVisibilityObserver'
 import { VisibilityObserver } from '@/services/createVisibilityObserver'
 import { Url } from '@/types/url'
 import { RouterPushOptions } from '@/types/routerPush'
+import { RouterLink } from '@/main'
```

Comment:
Eventually, we'll want to update all these tests to use the factories themselves. But continuing to test the global assets for this PR.